database/things/python.pldb

```diff
@@ -267,6 +267,7 @@ stackOverflowSurvey
   medianSalary 59454
   fans 34929
   percentageUsing 0.48
+annualReports https://www.python.org/psf-landing/
```

Comment:
great start! this is probably good enough. in the future we may want to link to the actual annual reports by year, but this gets us 80% of the value.